src/governance-vote.cpp

```diff
@@ -11,40 +11,42 @@
 
 std::string CGovernanceVoting::ConvertOutcomeToString(vote_outcome_enum_t nOutcome)
 {
-    switch(nOutcome)
-    {
-        case VOTE_OUTCOME_NONE:
-            return "NONE"; break;
-        case VOTE_OUTCOME_YES:
-            return "YES"; break;
-        case VOTE_OUTCOME_NO:
-            return "NO"; break;
-        case VOTE_OUTCOME_ABSTAIN:
-            return "ABSTAIN"; break;
+    switch (nOutcome) {
+    case VOTE_OUTCOME_NONE:
+        return "NONE";
+        break;
+    case VOTE_OUTCOME_YES:
+        return "YES";
+        break;
+    case VOTE_OUTCOME_NO:
+        return "NO";
+        break;
+    case VOTE_OUTCOME_ABSTAIN:
+        return "ABSTAIN";
+        break;
     }
     return "error";
 }
 
 std::string CGovernanceVoting::ConvertSignalToString(vote_signal_enum_t nSignal)
 {
     std::string strReturn = "NONE";
-    switch(nSignal)
-    {
-        case VOTE_SIGNAL_NONE:
-            strReturn = "NONE";
-            break;
-        case VOTE_SIGNAL_FUNDING:
-            strReturn = "FUNDING";
-            break;
-        case VOTE_SIGNAL_VALID:
-            strReturn = "VALID";
-            break;
-        case VOTE_SIGNAL_DELETE:
-            strReturn = "DELETE";
-            break;
-        case VOTE_SIGNAL_ENDORSED:
-            strReturn = "ENDORSED";
-            break;
+    switch (nSignal) {
+    case VOTE_SIGNAL_NONE:
+        strReturn = "NONE";
```

Comment:
Sigh... Two similar functions, two different approaches... And both could actually be refactored/simplified to work the way `ConvertVoteSignal` does it.